src/charm.py

```diff
@@ -575,6 +666,40 @@ def _build_layer(self) -> Layer:
 
         return layer
 
+    def _build_replication(self, primary: bool) -> Layer:
```

Comment:
Would this increase clarity?

```suggestion
    def _build_litestream_layer(self, primary: bool) -> Layer:
```

Reply:
Nah, it's overly specific given that we might want to switch out our replication component at some point. I'd suggest keeping it as is.